include/pmacc/simulationControl/SimulationHelper.hpp

```diff
@@ -331,7 +356,9 @@ namespace pmacc
                 ("checkpoint.restart.step", po::value<int32_t>(&restartStep),
                  "Checkpoint step to restart from")
                 ("checkpoint.period", po::value<std::string>(&checkpointPeriod),
-                 "Period for checkpoint creation")
+                 "Period for checkpoint creation [interval(s) based on steps]")
+                ("checkpoint.timePeriod", po::value<std::uint64_t>(&checkpointTimePeriod)->default_value(0),
+                 "Time periodic checkpoint creation [period in minutes]")
```

Comment:
I think there should be no default value for this flag. Rather, flag itself should be optional, and I think it is already. So technically perhaps it means to remove default value from here, and add a check somewhere in the internal code?

(Also, imo 0 is not a good default value for periods in general as period == 0 is not defined)

Reply:
I added to the variable that zero is disabling time depended checkpointing but not to the doc string because I assume the user will not try zero at all.